src/copycat.test.ts

```diff
@@ -84,6 +108,42 @@ test('generated values', () => {
         "Carroll",
         "Wunsch",
       ],
+      "postalAddress": Array [
+        "932 Krista Union, Santa Fe 8316, Maldives",
+        "655 Maudie Stravenue, Caldwell 5338, Paraguay",
+        "750 Nienow Hill, Stamford 5115, Haiti",
+        "809 Florencio View, West Covina 5836, Bhutan",
+        "237 Earnestine Inlet, Barnstable Town 7943, Israel",
+        "355 Conn Run, La Mirada 3381, Israel",
+        "821 Margaret Skyway, Conway 8574, Bahrain",
+        "96 Leuschke Prairie, Racine 7319, Barbados",
+        "71 Hahn Club, South Gate 5660, Cyprus",
+        "305 Roob Trail, Costa Mesa 5708, Macao",
```

Comment:
These values resemble addresses, but are far from "correct":
* If the cities actually exist in the country referred to in the address, that is purely by chance
* The address format doesn't change depending on the convention for that country
* The zip codes do not change depending on the convention for that country

I'm hoping this is "good enough" still. Faker get around this by not exposing any method to generate a full address, but it seems like such a common use case that I think it makes sense to provide it. If people want something more specific and correct, they can make use of the individual parts we expose (e.g. `city`).